scripts/shared.ts

```diff
@@ -18,31 +30,29 @@ export async function completion(
     stream = false,
     stop = ['</s>'],
     apiKey = key,
-    quiet = false
+    quiet = false,
+    container = defaultContainer
   } = {}
 ) {
-  if (!url || !apiKey) {
-    throw new Error('Missing url or key');
-  }
-
   if (!quiet) {
     console.info(`Calling ${url} with model ${model}, stream: ${stream}`);
   }
 
-  const bodyPayload: Record<string, unknown> = {
+  let bodyPayload: Record<string, unknown> = {
     id: Math.random().toString(36).substring(7),
     prompt,
     model,
     params: { max_tokens, stop },
     stream
   };
 
-  const p = await fetch(url, {
+  if (container) {
+    bodyPayload['runner'] = { container };
+  }
+
+  const p = await fetch(getApiUrl(''), {
```

Comment:
I left the completion endpoint at the root so it's not a breaking change, but seems like the completion endpoint should be under e.g. `/completion` instead--what do you think?

If it had its own endpoint instead of the root, default argument would probably be more confusing?

Reply:
That sounds more consistent long term to me. I think we should test on dev to see if it would replace the old deployment URL. If it does, we might need to devise a migration strategy for or-web project.

Reply:
Verified locally and it keeps the same URL thankfully